src/mavsdk/core/mavlink_command_sender.cpp

```diff
@@ -158,6 +169,7 @@ void MavlinkCommandSender::receive_command_ack(mavlink_message_t message)
         // Currently, the gripper/winch sends the ack with source sysid/compid 1/1 instead of 1/169.
         // Until that's fixed, we ignore the component ID for any commands going to the winch.
         const bool compid_exception =
+            (work->identification.target_component_id == MAV_COMP_ID_ALL) ||
```

Comment:
I doubt this one is required because MAVSDK doesn't send out commands to component ID 0, unless you changed that.

Reply:
Yeah it should not be needed then [c97a703](https://github.com/mavlink/MAVSDK/pull/2251/commits/c97a7036d101cbb0b23ed54ffb979a5b21f1d7f9)